packages/components/storybook/README.md

```diff
@@ -0,0 +1,11 @@
+
+# Storybook 
+
+Storybook is an open-source tool that provides a sandbox to develop and visualize components in isolation. See the [Storybookjs site](https://storybook.js.org/) for more information about the tool. 
+
+The Gutenberg project uses Storybook to view and work the the `@wordpress/components` package.
```

Comment:
Should it be `with the` instead of `the the`?

Reply:
:+1: